File: CS/WindowsFormsApplication88/CustomInnerColorPickControlViewInfo.cs
using DevExpress.XtraEditors;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using WindowsFormsApplication88.Properties;

namespace WindowsFormsApplication88 {
    public class CustomInnerColorPickControlViewInfo : InnerColorPickControlViewInfo {
        public CustomInnerColorPickControlViewInfo(InnerColorPickControl owner) : base(owner) {

        }
        protected override Image LoadMoreButtonGlyph() {
            return GetLoadModeButtonGlyph();
            //return base.LoadMoreButtonGlyph();
        }
        internal Image GetLoadModeButtonGlyph() {
            return Resources.pictureshapefillcolor_32x32;
        }
    }
}

File: VB/WindowsFormsApplication88/CustomInnerColorPickControlViewInfo.vb
Imports DevExpress.XtraEditors
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication88
    Public Class CustomInnerColorPickControlViewInfo
        Inherits InnerColorPickControlViewInfo

        Public Sub New(ByVal owner As InnerColorPickControl)
            MyBase.New(owner)

        End Sub
        Protected Overrides Function LoadMoreButtonGlyph() As Image
            Return GetLoadModeButtonGlyph()
            'return base.LoadMoreButtonGlyph();
        End Function
        Friend Function GetLoadModeButtonGlyph() As Image
            Return My.Resources.pictureshapefillcolor_32x32
        End Function
    End Class
End Namespace
